testsuite/features/step_definitions/common_steps.rb

```diff
@@ -72,9 +72,7 @@
 end
 
 Then(/^the system ID for "([^"]*)" should be correct$/) do |host|
-  $api_test.auth.login('admin', 'admin')
   client_id = $api_test.system.search_by_name(get_system_name(host)).first['id']
-  $api_test.auth.logout
```

Comment:
Why do you remove that part? Same with this part in in the file(s) below.

Reply:
Because in the rest of the test framework we are using a login and a logout at the beginning and end of a feature, or wherever we need them, but triggering them from the .feature side.
So, I tried to be consistent here too.